snapcraft/_store.py

```diff
@@ -336,6 +339,10 @@ def push(snap_filename, release_channels=None):
         logger.info('Uploaded {!r}'.format(snap_name))
     tracker.raise_for_code()
 
+    if os.environ.get('DELTA_UPLOADS_EXPERIMENTAL') and 'revision' in result:
```

Comment:
This may be defensive programming but isn't `revision` always supposed to be in the result and if it isn't it is because it did not pass review for which `tracker.raise_for_code` would have raised an exception by then.